php-classes/Slate/Progress/Narratives/Report.php

```diff
@@ -71,4 +71,85 @@ class Report extends \VersionedRecord
             'class' => \Slate\Term::class
         ]
     ];
-}
\ No newline at end of file
+
+    public static $searchConditions = array(
+        'narrativeID' => array(
+            'qualifiers' => array('narrativeid')
+            ,'points' => 2
+            ,'sql' => 'ID=%u'
```

Comment:
@BillClinton leading commas aren't used anymore and this file was already cleaned up